checker.js

```diff
@@ -0,0 +1,175 @@
+var fs         = require('fs')
+var Table      = require('cli-table');
+var colors     = require('colors');
+var path       = require('path');
+var Superagent = require('superagent');
+var readline   = require('readline-sync');
+
+var LIST_MODULES_URL = 'https://webtask.it.auth0.com/api/run/wt-tehsis-gmail_com-1?key=eyJhbGciOiJIUzI1NiIsImtpZCI6IjIifQ.eyJqdGkiOiJmZGZiOWU2MjQ0YjQ0YWYyYjc2YzAwNGU1NjgwOGIxNCIsImlhdCI6MTQzMDMyNjc4MiwiY2EiOlsiZDQ3ZDNiMzRkMmI3NGEwZDljYzgwOTg3OGQ3MWQ4Y2QiXSwiZGQiOjAsInVybCI6Imh0dHA6Ly90ZWhzaXMuZ2l0aHViLmlvL3dlYnRhc2tpby1jYW5pcmVxdWlyZS90YXNrcy9saXN0X21vZHVsZXMuanMiLCJ0ZW4iOiIvXnd0LXRlaHNpcy1nbWFpbF9jb20tWzAtMV0kLyJ9.MJqAB9mgs57tQTWtRuZRj6NCbzXxZcXCASYGISk3Q6c';
+var pkg = path.join(process.cwd(), 'package.json');
```

Comment:
This won't work for situations where the user puts their webtask entry-points in a subdirectory of the project.

Consider extracting this logic into its own function to be used here as well: https://github.com/auth0/webtask-bundle/blob/babel/index.js#L79-L98

That just gives the directory. The `package.json` still needs to be appended: https://github.com/auth0/webtask-bundle/blob/babel/index.js#L107.